neointerface/neointerface.py

```diff
@@ -219,13 +221,13 @@ def query(self, q: str, params=None, return_type: str = 'data', convert_dates: b
                 result_data = result.data()
                 if convert_dates:
                     self.update_values(source=result_data)
-                result_1dict = []  # best-guess-merging all the results into 1 dict
+                flattened_dictionaries = []
                 for r in result_data:
-                    dct = {}
-                    for k, i in r.items():
-                        dct = {**dct, **(i if isinstance(i, dict) else {k: i})}
-                    result_1dict.append(dct)
-                return pd.DataFrame(result_1dict)
+                    if 'map' in r:
+                        # usually return a dict with the label 'map' in a cypher query. Remove that extra level here
+                        r = r['map']
+                    flattened_dictionaries.append(self.flatten(r))  # pd.json_normalise() might be a better choice
+                return pd.DataFrame(flattened_dictionaries)
```

Comment:
test_query_pd fails - please fix (I don't think we need 'map.; prefix in column name in that case @WillMcDGSK